EMULATOR/EMULATOR.C

```diff
@@ -237,19 +237,21 @@ int Emulator_InitialiseGameVariables()
 	extern unsigned long* GadwOrderingTables_V2;
 	extern unsigned long* terminator;
 
+// MAGICAL CONST 231604 is equal to (5128 * 4) + (52260 * 4) + (512 * 4) + 4 WHY?
```

Comment:
I'd love to give this a name somewhere. It's allocating this much virtual memory, but it seems a very specific amount

Reply:
See:
5128 * 4 is clear size of GadwOrderingTables.
52260 * 4 is clear size of GadwPolygonBuffers.
512 * 4 is clear size of GadwOrderingTables_V2.
+ 4 is clear size of terminator.